functions/__sf_section_battery.fish

```diff
@@ -73,7 +73,7 @@ function __sf_section_battery -d "Displays battery symbol and charge"
 	end
 
 	 # Remove trailing % and symbols for comparison
-	set battery_percent (echo $battery_percent | tr -d "%[,;]")
+	set battery_percent (echo $battery_percent | string trim --chars=%[,;])
```

Comment:
Use fish string manipulation utils to ensure compatibility with unix emulators that may not have `tr`.